api/src/oscc.c

```diff
@@ -26,73 +20,86 @@ static int can_socket = -1;
 
 oscc_result_t oscc_open( unsigned int channel )
 {
-    oscc_result_t ret = OSCC_ERROR;
+    oscc_result_t result = OSCC_ERROR;
 
-    char buffer[16];
 
-    snprintf( buffer, 16, "can%1d", channel );
+    char can_string_buffer[16];
 
-    printf( "Opening CAN channel: %s\n", buffer );
+    snprintf( can_string_buffer, 16, "can%1d", channel );
```

Comment:
Should the conversion specifier be `%1u` since `channel` is type`unsigned int`?

Reply:
It should definitely be `%u`, I'm surprised there were no compiler warnings about that. 567c5b8

Also, the field width has been removed, it's unnecessary. be8ca94